### ISSUE TITLE
Use ordinal headings

### ISSUE THREAD

**Issue by ciaoaaron** [r-moyer/Desktop-Site#3]
https://github.com/r-moyer/Desktop-Site/blob/67342bca6c8e02a92d6484a4df0785436138f479/index.html#L25

Headings should be in ordinal order to make sure they are exposed correctly to screen readers.

First fix [this](https://github.com/r-moyer/Desktop-Site/issues/7), and then make sure all your headings are in the right order.
